flocx_market/tests/base_test.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+from flocx_market.cmd.app import application
+from flocx_market.db.orm import orm
+
+
+class BaseTest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # ensure db exists
+        application.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://gpdwpyrw:UCzgXashsqnX99ASMK2SxZK8d4tbEUa6@raja.db.elephantsql.com:5432/gpdwpyrw'
```

Comment:
Integration testing is a great idea!  But I don't think we should have any tests that rely on an external service. A typical way of handling integration tests is to have the CI environment set up whatever services are necessary for testing, and then pass in the necessary configuration to the tests.  I would suggest simply removing everything but the unit tests from this pull request, and then implement integration tests separately.